app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = list()
+
+    def __init__(
+            self, name: str, health: int = 100, hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
+
+    def __repr__(self) -> dict:
+        animal = (f"{{Name: {self.name}, Health: {self.health},"
+                  f" Hidden: {self.hidden}}}")
+        return animal
```

Comment:
The `__repr__` method should return a string representation of the object, not a dictionary. The current implementation returns a string, so the method name might be misleading. Consider renaming the method or adjusting the return type if a dictionary is intended.